content/en/logs/processing/parsing.md

```diff
@@ -495,6 +496,63 @@ rule %{data::xml}
 * If the XML contains tags that have both an attribute and a sting value between the two tags, a `value` attribute is generated. For example: `<title lang="en">Harry Potter</title>` is converted to `{"title": {"lang": "en", "value": "Harry Potter" } }`
 * Repeated tags are automatically converted to arrays. For example: `<bookstore><book>Harry Potter</book><book>Everyday Italian</book></bookstore>` is converted to `{ "bookstore": { "book": [ "Harry Potter", "Everyday Italian" ] } }`
 
+### Parsing CSV
+
+Use the **csv** filter to more-easily map strings to attributes when separated by a given character (`,` by default).
+
+The csv filter is defined as `csv(headers[, separator[, quotingcharacter]])` where:
+
+* `headers`: defines the keys name separated by `,`. Keys names must start with alphabetical character and can contain any alphanumirical character in addition to `-`, `_`, `@` and `$`.
+* `separator`: defines separators used to separate the different values. Only one character is accepted. Default: `,` . Note use `tab` to represent the tabulation character.
+* `quotingcharacter`: defines the quoting character. Only one character is accepted. Default: `"` 
+
+**Note**:
+
+* Values containing a separator character must but quoted.
+* Quoted Values containing a quoting character must be escaped with a quoting characters. ex: `""` represent `"`.
+* If the log doesn't contain the same number of value as the number of keys in the header, the csv parser will match the first ones. 
+
+
+**Log**:
+
+```text
+John,Doe,120 jefferson st.,Riverside, NJ, 08075
```

Comment:
Although this example correctly shows the current behavior, I would avoid the whitespace after the last comma, as this is a specific case where our CSV parser has potentially questionable behavior (we strip whitespace outside of quotes, which is different than a lot of other CSV parsers). Having a more "standard" example CSV here allows us to avoid documenting this behavior for now.

```suggestion
John,Doe,120 jefferson st.,Riverside,NJ,08075
```

Reply:
Added in the Notes and in the first example.